<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana3</class>
 <widget class="QWidget" name="ventana3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="botonok">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>100</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="usuario_line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>651</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLineEdit {
    background-color: transparent;
    border: none;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="contrasena_line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>651</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLineEdit {
    background-color: transparent;
    border: none;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="referencia_line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>651</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLineEdit {
    background-color: transparent;
    border: none;
}
</string>
   </property>
  </widget>
  <widget class="QTableView" name="tabla">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>941</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_eliminar">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>170</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1001</width>
     <height>591</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../PROYECTO FINAL/imagenes/contrase√±as.jpg</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>botonok</zorder>
  <zorder>usuario_line</zorder>
  <zorder>contrasena_line</zorder>
  <zorder>referencia_line</zorder>
  <zorder>tabla</zorder>
  <zorder>boton_eliminar</zorder>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>add_data()</slot>
  <slot>eliminar()</slot>
 </slots>
</ui>
